eng/versioning/set_versions.py

```diff
@@ -182,7 +182,7 @@ def update_versions_file_for_nightly_devops(build_type, build_qualifier, artifac
                         # if the module is 1.2.3-beta.x, strip off everthing after the '-' and add the qualifier
                         module.dependency = module.dependency[:module.dependency.rfind('-') + 1] + unreleased_build_qualifier
                     else:
-                        module.dependency = module.dependency += '-' + unreleased_build_qualifier
+                        module.dependency += '-' + unreleased_build_qualifier
```

Comment:
gah...this snuck in there because I'd accidentally submitted a mangled version_client.txt and reverting that also killed the last change to this file which was removal of this syntax error.